runtime_patch.go

```diff
@@ -60,6 +60,99 @@ func updateMagicValue(file *ast.File, magicValue uint32) {
 	}
 }
 
+// updateEntryOffset adds xor encryption for funcInfo.entryoff
```

Comment:
could you document what is the algorithm, and how the final offsets aren't easy to deobfuscate quickly?

Reply:
also please document that it's cheap and shouldn't have noticeable overhead, assuming that is the case.